src/main/java/seedu/sudohr/logic/commands/eventCommands/DeleteEmployeeFromEventCommand.java

```diff
@@ -0,0 +1,89 @@
+package seedu.sudohr.logic.commands.eventcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.sudohr.commons.core.index.Index;
+import seedu.sudohr.logic.commands.Command;
+import seedu.sudohr.logic.commands.CommandResult;
+import seedu.sudohr.logic.commands.exceptions.CommandException;
+import seedu.sudohr.model.Model;
+import seedu.sudohr.model.events.Event;
+import seedu.sudohr.model.events.EventContainsEmployeePredicate;
+import seedu.sudohr.model.person.Person;
+
+/**
+ * Deletes a person from a specific event in sudohr book.
+ */
+public class DeleteEmployeeFromEventCommand extends Command {
+    public static final String COMMAND_WORD = "deleteEmployeeEvent";
+
+    public static final String MESSAGE_PERSON_DOES_NOT_EXIST = "This user does not exists in the event in sudohr book";
+
+    public static final String MESSAGE_SUCCESS = "person %1$s is deleted from %2$s";
+
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index is invalid";
+
+    public static final String MESSAGE_INVALID_EVENT_DISPLAYED_INDEX = "The event index is invalid";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a employee from event in the sudohr book. ";
+
+    private final Index eventToDeleteIndex;
+    private final Index personToDeleteIndex;
+
+    /**
+     * Creates an DeleteEmployeeFromEventCommand to delete the person at specified
+     * {@code personIndex} from the event at the specified {@code eventIndex}
+     */
+    public DeleteEmployeeFromEventCommand(Index personIndex, Index eventIndex) {
```

Comment:
After Andre's merge and the green light to pull (I'll update the group), can you also update the personIndex to Employee ID